### PR TITLE
Update Academy of Management Style

### PR DESCRIPTION
**Pull request by bwiernik**
Updated according to current style guide.

1. Fix title case of journal titles
2. Consistently format titles of items and containers across item types
3. Fix formatting of genre, publisher, and publisher place as best as
CSL can handle
4. Adjusted display of URL to only display for non-print references and to prefer issue date over access date

### CODE REVIEW

File: academy-of-management-review.csl [hunk 1]
```diff
@@ -234,16 +252,23 @@
           <group>
             <text macro="container-contributors"/>
             <text macro="secondary-contributors"/>
-            <text variable="container-title" font-style="italic" font-weight="bold"/>
+            <choose>
+              <if type="article-journal article-magazine article-newspaper paper-conference" match="any">
```
Comment (adam3smith) [inline academy-of-management-review.csl]: are you basing the title case for conference proceedings on more than the one example in the guide for authors? That seems like a very odd rule to me.

**Comment by adam3smith** [comment]
could you also update the documentation link to http://aom.org/uploadedFiles/Publications/AMR/AMR%20Style%20Guide%202014.pdf

File: academy-of-management-review.csl (hunk 1 above)
Comment (bwiernik) [inline academy-of-management-review.csl]: For this journal, most conference proceedings that are cited will be published in journal-like format (specifically the Academy of Management Proceedings), so the proceedings title should be capitalized like a journal title. I'm not sure the best way to handle both formats of proceedings correctly (as book-like proceedings should be in Title case). Would it work to test for an editor (as most journal-like proceedings don't have editors that are cited)?

Comment (adam3smith) [inline academy-of-management-review.csl]: hmmm. And you'd actually expect something like Academy of Management Proceedings to appear in databases in sentence case? I'm fine testing for editor if you want to go that way, it just seems like a lot of complexity for few cases and I always worry that those types of rules may confuse end users. But as I said, if you want to put it in, I won't object. 
Please add a comment in the code explaining that, though, so later authors can make sense of it.

**Comment by rmzelle** [comment]
@bwiernik, can you make the requested changes?

**Comment by bwiernik** [comment]
Yes, I will get to them in the next few days

File: academy-of-management-review.csl [hunk 2]
```diff
@@ -82,30 +86,31 @@
...
+            <text variable="URL"/>
+            <choose>
+              <if variable="issued" match="none">
+                <group prefix=", ">
+                  <date variable="accessed">
+                    <date-part name="month" suffix=" "/>
+                    <date-part name="day" suffix=", "/>
+                    <date-part name="year"/>
+                  </date>
+                </group>
```
Comment (zuphilip) [inline academy-of-management-review.csl]: You don't need the `group` here, the `prefix` can go into the `date`, cf. http://citationstyles.org/downloads/specification.html#affixes .

**Comment by bwiernik** [comment]
I've made the requested changes. Also fixed the formatting of book volumes and presented-at. Updated the documentation for Academy of Management Journal. 

I dropped the inclusion of conference papers in the bibliography title-casing on the assumption that the proceedings title will always be in Title Case in databases and that no styles require it to be in sentence case.

File: academy-of-management-review.csl [hunk 3]
```diff
@@ -131,8 +146,19 @@
   <macro name="event">
     <choose>
       <if variable="event">
-        <text term="presented at" text-case="capitalize-first" suffix=" "/>
-        <text variable="event"/>
+        <choose>
```
Comment (adam3smith) [inline academy-of-management-review.csl]: would you mind fixing the indenting here? Everything else looks good.

Comment (bwiernik) [inline academy-of-management-review.csl]: Fixed. Sorry about that.

**Comment by adam3smith** [comment]
thanks!

**Pull request merged**